### ISSUE TITLE
[Bug] 2.0 - Using Tag/Element name caused unsettable properties.

### ISSUE THREAD

**Issue by NathanaelA** [NativeScript/theme#157]
I am unsure if you are aware; but because themes 2.0 uses Element/Tag names (i.e. <Button> will be styled with the "Button" css style rather than requiring me to tag it with style="btn" like in v1.0) that any styles set on "Button" are now **unsettable**.  

NS (https://github.com/NativeScript/NativeScript/issues/628) does not have a "inherit", "initial", "unset" or even "!important" support.   Which means there is no way to reset back a control's item back to defaults when using 2.0...   This makes themes in 2.0 dangerous and can break many things, or require adding additional code to try and reset things to default....  ;-(

**Comment by bundyo** [comment]
Yes, I even have an inherit branch in {N} repo. I hope to get this fixed soon.

**Comment by NathanaelA** [comment]
@bundyo - Awesome news!     So are you are going to handle the "initial" and "unset" which are really needed to reset back to defaults?
